frame/system/src/lib.rs

```diff
@@ -518,6 +520,16 @@ decl_module! {
 			ensure_signed(origin)?;
 		}
 
+		/// A noop
+		#[weight = 0]
+		fn ensure_signed(origin) {
```

Comment:
New transaction for test here. Only does `ensure_signed(origin);`

Reply:
I think this is not different in any way to `remark` right? or is it?

Reply:
In practice I don't know, but I was asked to remove the input. I also tried without `ensure_signed` and it was a margin of error difference < 3%.

Reply:
yeah, my guess is that with remark, you would get the exact same result. Input of `vec![]` should do nothing. Literally.

Reply:
I suggest you try it because it would make more sense in terms of merging this PR if that is an option :)

Reply:
This started as an experiment to get some quick numbers (for different purposes) and will need to be tidied up a lot if we want to merge something. We had a meeting a while ago and when discussed I noted that this particular benchmark was not intended to be merged - but just a one-off thing; that was a while ago though and obviously I can see the benefit to merging something so that we can repeat as changes occur. 

However I need to spend a bit more time working out some details. I'd also like to split the profiler code from the benchmarking code into 2 separate PRs. I'm already working on it, so hopefully wont be long until we have a better version of this.